src/monexamine.cpp

```diff
@@ -263,6 +266,17 @@ int monexamine::pet_armor_pos( monster &z )
     return g->u.get_item_position( loc.get_item() );
 }
 
+int monexamine::tack_pos()
+{
+    auto filter = []( const item & it ) {
+        return it.has_flag( "TACK" );
+    };
+
+    item_location loc = game_menus::inv::titled_filter_menu( filter, g->u, _( "Tack" ) );
+
+    return g->u.get_item_position( loc.get_item() );
```

Comment:
If `loc.get_item()` yields a null pointer (which is possible), you get undefined behaviour inside of `Character::get_item_position` as it dereferences that pointer. You need to check `loc` for being valid before that.